conventional-changelog-lighthouse/templates/commit.hbs

```diff
@@ -0,0 +1,29 @@
+{{!--
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+--}}
+* {{#if scope}}{{scope}}: {{/if~}}
+  {{#if message ~}}
+    {{~message~}}
+  {{~else~}}
+    {{~header~}}
+  {{~/if ~}}
+{{~!-- commit hash --}}
+{{~#if @root.linkReferences~}}
+  {{~#if PR ~}}
+    ([#{{PR}}](
+      {{~#if @root.host}}{{~@root.host}}/{{/if~}}
+      {{~#if @root.owner ~}}{{@root.owner}}/{{/if~}}
+      {{~@root.repository}}/pull/{{PR}}))
+  {{~else~}}
+    ([{{hash}}](
+      {{~#if @root.host}}{{~@root.host}}/{{/if~}}
+      {{~#if @root.owner ~}}{{@root.owner}}/{{/if~}}
+      {{~@root.repository}}/{{@root.commit}}/{{hash}}))
+  {{~/if~}}
+{{~else~}}
+  {{~hash~}}
```

Comment:
I think we need to preserve a single space here, the formatting looks off compared to ones with a PR link

![image](https://user-images.githubusercontent.com/2301202/32465423-4484cc84-c2f8-11e7-8d35-977f78bf8567.png)